wicket-core/src/main/java/org/apache/wicket/ajax/AbstractDefaultAjaxBehavior.java

```diff
@@ -233,12 +246,6 @@ protected final CharSequence renderAjaxAttributes(final Component component,
 				attributesJson.put(AjaxAttributeName.MARKUP_ID.jsonName(), componentId);
 			}
 
```

Comment:
I'm just curious...why have you moved the 5 lines below at the beginning of the method (line 223)?